arch/xtensa/src/common/xtensa_int_handlers.S

```diff
@@ -197,6 +197,13 @@ _xtensa_level1_handler:
 
 	call0	_xtensa_context_save
 
+	/* Overwrite the PS saved by the exception entry. */
```

Comment:
This is clearing the PS.EXCM, right? If so, why?

`CINTLEVEL ← max(PS.EXCM∗EXCMLEVEL,PS.INTLEVEL)`

So, clearing the PS.EXCM would enable other interrupts to be served, but NuttX doesn't support nested interrupts.

Reply:
This clearing the saved PS.EXCM, not current PS.
I have added the explain here:
https://github.com/apache/nuttx/pull/14672#issuecomment-2553126081